### PR TITLE
[FX] Sync enhancement done internally at Meta

### PR DESCRIPTION
**Pull request by yinghai**
91196dedd10ed4061bef719b01fe917192a6fc17 Shiyan Deng <dsy842974287@fb.com> fix EngineHolder
608acce060d91ac93099f59e6165d43ffd79d50a Shiyan Deng <dsy842974287@fb.com> delete unused file
f88cb1566ae1c72634668fa26fadc3ef368b9727 Alex Beloi <alexbeloi@fb.com> [fx] add common_subexpression_elimination graph opt
b079de89bce912c29284809722cae63cb185c481 Yinghai Lu <yinghai@fb.com> handle list of integers in SliceOp check
e8f8dbda72625872a2657cdfd09dd59209f98090 Shiyan Deng <dsy842974287@fb.com> Add a pass to eliminate unsqueeze + cat + getitem pattern
675b28120c432c63bf4d451a4479b2376f73201d Kefei Lu <kefeilu@fb.com> Back out "[fx] add common_subexpression_elimination graph opt"
a3fb38bb6193b86a0ef0795a9104c073f5044598 Shiyan Deng <dsy842974287@fb.com> Back out "Back out "[fx] add common_subexpression_elimination graph opt""
0ea2b5782d70ed07b2158a96a45082aba68c55f6 Andrew Or <andrewor@fb.com> [Quant][fx][bc-breaking] Replace is_reference with convert_to_reference
2e4384397e73e99a3d56303864dc09e23420dda1 Shreyansh Prajapati <shreyanshp@fb.com> Test dynamic shape suport for acc_ops.sigmoid
78f7505cbe18850f43261b5f92e67262127e229e Mor Tzur <mortzur@fb.com> uncomment accidentally commented out acc_op tests
a7c0210a5c6468d978fc55a541b08da4e1782df7 Shreyansh Prajapati <shreyanshp@fb.com> Test dynamic shape support for acc_ops.avgpool and error message change
8aab78b134c778bcafe56f0fd5baa9bf51d7fef9 Shreyansh Prajapati <shreyanshp@fb.com> Test dynamic shape support for acc_ops.any
0591b671542c186a257152f8ed82aecbd2322feb Shreyansh Prajapati <shreyanshp@fb.com> Test dynamic shape support for acc_ops.avgpool
5d36837498f72f5ad3432eecf3f4e4091ac23071 Shreyansh Prajapati <shreyanshp@fb.com> Test dynamic shape support for acc_ops.batchnorm
9a3134742872040b31a70bf216c81de5dcd82c45 Shreyansh Prajapati <shreyanshp@fb.com> Test dynamic shape support for acc_ops.binary_ops
4802100968d39418d6d1ce75c0407b3344e5cfd9 Shreyansh Prajapati <shreyanshp@fb.com> Test dynamic shape support for acc_ops.as_strided
6ccf77223a40e16d635d453b6342658bdc44062e Shreyansh Prajapati <shreyanshp@fb.com> Test dynamic shape support for acc_ops.cat
0dd29ac935705926cc29bf03f5da922d93adae17 Shreyansh Prajapati <shreyanshp@fb.com> Test dynamic shape support for acc_ops.chunk
81737ebcecb0a57a683971d9052b3f3262030d20 Shreyansh Prajapati <shreyanshp@fb.com> Added test case for testing dynamic shape support for acc_ops.clamp
0c2604c6db94309c8a76de00e8c69ce21d2114a8 Yinghai Lu <yinghai@fb.com> [fx2trt] Fix constant tensor conversion dtype bug

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant and/or add your own.

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)
- This change requires a documentation update

# Checklist:

- [x] My code follows the style guidelines of this project (You can use the linters)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas and hacks
- [x] I have made corresponding changes to the documentation
- [x] I have added tests to verify my fix or my feature
- [x] New and existing unit tests pass locally with my changes
- [x] I have added the relevant labels to my PR in so that relevant reviewers are notified

### CODE REVIEW

**Review by frank-wei — APPROVED** [review]
lgtm!

**Comment by yinghai** [comment]
Wait. CircleCI failed. It seems that we need to install tabulate python package?

**Comment by frank-wei** [comment]
yes, I noticed that. Fix in another PR https://github.com/pytorch/TensorRT/pull/1163
